syntax = "proto3";  //protobuf版本
package fixbug;  //类似于C++的命名空间


//选项
option cc_generic_services =true;


message Resultcode 
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

message LoginRequest 
{
    bytes name=1;
    bytes pwd=2;
}

message  LoginResponse 
{
    Resultcode result = 1;
    bool success = 2;
}


service  UserServiceRpc
{
    rpc Login (LoginRequest) returns (LoginResponse);
}